engines/gob/detection/tables_adi2.h

```diff
@@ -127,7 +127,7 @@
 	"adi2.stk", "ediintro.tot", 0
 },
 {
-	{ // 1994 CD version
+	{ // 1994 CD version - Supplied by BJNFNE in commit: 238df35
```

Comment:
Specifiying the commit is useless as it can always be found through git blame.

Reply:
So i should write then instead Supplied by BJNFNE as example? If it was added over an commit?